core/domain/exp_services_test.py

```diff
@@ -2529,7 +2499,7 @@ class ZipFileExportUnitTests(ExplorationServicesUnitTests):
 auto_tts_enabled: false
 blurb: ''
 category: Algebra
-correctness_feedback_enabled: false
+correctness_feedback_enabled: true
```

Comment:
The only place to change this to True is in the storage layer, since we want to be careful about how we handle existing storage models..

In all other places it should be removed entirely.

Reply:
As mentioned above, next_content_id_index is not deprecated.

Reply:
dropped the CMD as suggested below.